docs/Hosts/Spawn-Hosts.md

```diff
@@ -98,7 +98,7 @@ UI](https://spruce.mongodb.com/spawn/host) by pressing the "edit" button your ho
 In the Spruce UI, you can pick a sleep schedule for your host when creating the unexpirable host or editing an existing
 unexpirable host:
 
-![Sleep schedule](../images/set_sleep_schedule.png)
+_![Sleep schedule](../images/set_sleep_schedule.png)_
```

Comment:
Out of curiosity, what does these _ do to the image?

Reply:
oops i think that was just a mistake

Reply:
i'd love if it made the image italic though